docs/README.md

```diff
@@ -0,0 +1,42 @@
+# Slide Decks
```

Comment:
I love the idea of using markdown slides. I spent a bit of time researching this before I interviewed. I'm a bit unclear about what there is to review here. Will this be live soon (Ada-Developers-Academy.github.io/jump-start-live)?

Reply:
That's fair, I didn't explicitly call out why I'm making this change.

@CheezItMan and I had talked on Friday about the idea of using markdown slides in place of what we currently use, Google slides. This MR was a quick proof of concept to demo to see if something could be made to support the transition (I only ported the first couple of slides from day 1), and I wanted a couple of eyes on it to see if the solution presented is an acceptable one.

Reply:
I guessed at the URL, based on github pages syntax, but if this gets merged, then yes it should be live and viewable at http://Ada-Developers-Academy.github.io/jump-start-live (if not there, then whatever the github pages url would be)